interleave_rqcfilter.wdl

```diff
@@ -190,7 +190,7 @@ task rqcfilter{
             File stat = filename_stat
             File stat2 = filename_stat2
             File info_file = filename_reproduce
-            File filtered = glob("filtered/*fastq.gz")[0]
+            File filtered = glob("filtered/*.anqrpht.fastq.gz")[0]
```

Comment:
This does work but it would be better to remove the glob. We are only expecting a single file so there is no need for the glob.

Reply:
updated, run 84396